### PR TITLE
Add /stats Command

### PR DESCRIPTION
**Pull request by moreavy**
image
A command to list all kills, deaths and score of players together.

### CODE REVIEW

File: pvp_club/init.lua [hunk 1]
```diff
@@ -320,3 +320,26 @@ mt.register_on_respawnplayer(function (player)
     })  
     player:set_hp(100)
 end)
+
+minetest.register_chatcommand("skills", {
```
Comment (BrownHedgehog78) [inline pvp_club/init.lua]: I think better to change it to stats:

```suggestion
minetest.register_chatcommand("stats", {
```

File: pvp_club/init.lua [hunk 2]
```diff
@@ -320,3 +320,26 @@ mt.register_on_respawnplayer(function (player)
...
+            kills = ms:get_string(name.."kills")
+        end
+        if ms:get_string(name.."deaths") ~= (nil or "") then
+            deaths = ms:get_string(name.."deaths")
+        end
+        if ms:get_string(name.."score") ~= (nil or "") then
+            score = ms:get_string(name.."score")
+        end
+
+        minetest.chat_send_player(name, "Skills of "..minetest.colorize(PVP.team_color(name), name).." are:\nKills: "..kills.."\nDeaths: "..deaths.."\nSocre: "..score)
```
Comment (BrownHedgehog78) [inline pvp_club/init.lua]: And the second one:

```suggestion
        minetest.chat_send_player(name, "Stats of "..minetest.colorize(PVP.team_color(name), name).." are:\nKills: "..kills.."\nDeaths: "..deaths.."\nSocre: "..score)
```

**Review by moreavy — COMMENTED** [review]
Tested: Working Fine!

File: pvp_club/init.lua [hunk 3]
```diff
@@ -317,6 +321,52 @@ end)
...
+
+minetest.register_chatcommand("stats", {
+    privs = {
+        interact = true,
+    },
+    func = function (name, param)
+        if param ~= ("" or nil) then
+
+            if not table.includes(PVP.players, param) then
+                return true,"Invalid Player Name!"
```
Comment (BrownHedgehog78) [inline pvp_club/init.lua]:
```suggestion
                return true,"Invalid player name"
```

Comment (moreavy) [reply pvp_club/init.lua]:
```suggestion
                return true,"Invalid player name."
```

**Pull request merged**